### PR TITLE
Change ts() to E::ts()

### PR DESCRIPTION
**Pull request by lucky091588**

### CODE REVIEW

File: CRM/Eventbrite/DAO/EventbriteLink.php [hunk 1]
```diff
@@ -104,7 +104,7 @@ public static function &fields() {
         'id' => [
           'name' => 'id',
           'type' => CRM_Utils_Type::T_INT,
-          'description' => ts('Unique EventbriteLink ID'),
+          'description' => E::ts('Unique EventbriteLink ID'),
```
Comment (twomice) [inline CRM/Eventbrite/DAO/EventbriteLink.php]: Remember we don't want to bother editing DAO files; these files are managed by civix and our changes will be overwritten next time civix makes any change to the file. @lucky091588

File: CRM/Eventbrite/DAO/EventbriteLog.php [hunk 2]
```diff
@@ -75,7 +75,7 @@ public static function &fields() {
         'id' => [
           'name' => 'id',
           'type' => CRM_Utils_Type::T_INT,
-          'description' => ts('Unique EventbriteLog ID'),
+          'description' => E::ts('Unique EventbriteLog ID'),
```
Comment (twomice) [inline CRM/Eventbrite/DAO/EventbriteLog.php]: Same comment here about DAO files.

File: CRM/Eventbrite/DAO/EventbriteQueue.php [hunk 3]
```diff
@@ -75,7 +75,7 @@ public static function &fields() {
         'id' => [
           'name' => 'id',
           'type' => CRM_Utils_Type::T_INT,
-          'description' => ts('Unique EventbriteQueue ID'),
+          'description' => E::ts('Unique EventbriteQueue ID'),
```
Comment (twomice) [inline CRM/Eventbrite/DAO/EventbriteQueue.php]: Same comment here about DAO files.

File: CRM/Eventbrite/Upgrader/Base.php [hunk 4]
```diff
@@ -200,7 +200,7 @@ public function enqueuePendingRevisions(CRM_Queue_Queue $queue) {
     $currentRevision = $this->getCurrentRevision();
     foreach ($this->getRevisions() as $revision) {
       if ($revision > $currentRevision) {
-        $title = ts('Upgrade %1 to revision %2', array(
+        $title = E::ts('Upgrade %1 to revision %2', array(
```
Comment (twomice) [inline CRM/Eventbrite/Upgrader/Base.php]: Same comment here about civix-managed files.

**Review by twomice — COMMENTED** [review]
See comments about civix-managed files @lucky091588 Thanks.

**Comment by twomice** [comment]
Thanks @lucky091588 Can you squash this to one commit?

**Comment by lucky091588** [comment]
@twomice I already check this and there is not `E` error.

File: CRM/Eventbrite/Form/Manage/Event.php [hunk 5]
```diff
@@ -121,7 +121,7 @@ public function postProcess() {
       }
       $result = _eventbrite_civicrmapi('EventbriteLink', 'create', $apiParams);
     }
-    CRM_Core_Session::setStatus(ts('Settings have been saved.'), ts('Saved'), 'success');
+    CRM_Core_Session::setStatus(ts('Settings have been saved.'), E::ts('Saved'), 'success');
```
Comment (twomice) [inline CRM/Eventbrite/Form/Manage/Event.php]: @lucky091588 Looks like there's one more `ts` call here that needs the same treatment.

File: CRM/Eventbrite/Form/Manage/Field.php [hunk 6]
```diff
@@ -179,7 +179,7 @@ public function postProcess() {
       }
       $result = _eventbrite_civicrmapi('EventbriteLink', 'create', $apiParams);
     }
-    CRM_Core_Session::setStatus(ts('Settings have been saved.'), ts('Saved'), 'success');
+    CRM_Core_Session::setStatus(ts('Settings have been saved.'), E::ts('Saved'), 'success');
```
Comment (twomice) [inline CRM/Eventbrite/Form/Manage/Field.php]: @lucky091588 Also, looks like there's one more ts call here that needs the same treatment.

File: CRM/Eventbrite/Form/Manage/Tickettype.php [hunk 7]
```diff
@@ -141,7 +141,7 @@ public function postProcess() {
       }
       $result = _eventbrite_civicrmapi('EventbriteLink', 'create', $apiParams);
     }
-    CRM_Core_Session::setStatus(ts('Settings have been saved.'), ts('Saved'), 'success');
+    CRM_Core_Session::setStatus(ts('Settings have been saved.'), E::ts('Saved'), 'success');
```
Comment (twomice) [inline CRM/Eventbrite/Form/Manage/Tickettype.php]: @lucky091588 Again, looks like there's one more ts call here that needs the same treatment.

**Comment by lucky091588** [comment]
@twomice Done updating the files that I missed.

**Comment by twomice** [comment]
Merged, thanks @lucky091588

**Pull request merged**